### PR TITLE
Cloud Env Cluster Support

### PR DESCRIPTION
**Pull request by paulotruta**
Adds support for building and starting Edgebox in a multipass clustering setup powered by Traefik and HaProxy

TODO: Add proper method comments on the py script
TODO: Add instructions and requirements to the readme
MAYBE TODO: Add traefik setup from scratch?

### CODE REVIEW

File: scripts/setup.sh [hunk 1]
```diff
@@ -45,39 +50,106 @@ groupadd docker
...
+pip3 -v install docker==6.1.3
 pip3 -v install docker-compose
 pip3 -v install yq
 
 # Install cloudflared
 wget -q https://github.com/cloudflare/cloudflared/releases/download/2023.3.1/cloudflared-linux-$(printenv ARCH).deb
 sudo dpkg -i cloudflared-linux-$(printenv ARCH).deb
 
+# Install sshx
+curl -sSf https://sshx.io/get | sh
```
Comment (paulotruta) [inline scripts/setup.sh]: Closes #1

**Review by inverse — APPROVED** [review]
Seems legit :+1:

File: scripts/rebuild_proxies.py [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+        # [http.routers.adm-edgebox]
+        #     rule = "Host(`adm.edgebox.io`)"
+        #     service = "adm-edgebox-service"
+        #     priority = 2
+
+        # [http.services.adm-edgebox-service.loadBalancer]
+        #     [[http.services.adm-edgebox-service.loadBalancer.servers]]
+        #         url = "http://10.139.121.36:80"
+
+        traefik_generated_config += "[http.routers." + proxy['name'] + "-edgebox-wildcard]\n"
```
Comment (inverse) [inline scripts/rebuild_proxies.py:58]: Perhaps we could improve this with leveraging the toml package and writing in a nicer way.

See: https://www.geeksforgeeks.org/read-and-write-toml-files-using-python/

Comment (paulotruta) [reply scripts/rebuild_proxies.py:58]: Absolutely 🙌 looks way cleaner, thanks for the suggestion!

**Pull request merged**